src/HttpParser.jl

```diff
@@ -85,6 +85,34 @@ type ParserSettings
     on_message_complete_cb::Ptr{Void}
     on_chunk_header::Ptr{Void}
     on_chunk_complete::Ptr{Void}
+
+    function ParserSettings(
+      on_message_begin_cb::Ptr{Void} = Ptr{Void}(0),
+      on_url_cb::Ptr{Void} = Ptr{Void}(0),
+      on_status_complete_cb::Ptr{Void} = Ptr{Void}(0),
+      on_header_field_cb::Ptr{Void} = Ptr{Void}(0),
+      on_header_value_cb::Ptr{Void} = Ptr{Void}(0),
+      on_headers_complete_cb::Ptr{Void} = Ptr{Void}(0),
+      on_body_cb::Ptr{Void} = Ptr{Void}(0),
+      on_message_complete_cb::Ptr{Void} = Ptr{Void}(0),
+      on_chunk_header::Ptr{Void} = Ptr{Void}(0),
+      on_chunk_complete::Ptr{Void} = Ptr{Void}(0))
+        function dummy(parser)
+          0
```

Comment:
The definition of `HTTP_CB` is actually wrong. It should use `Cint` instead of `Int`.